src/he/ssl_ctx.h

```diff
@@ -470,6 +470,21 @@ bool he_ssl_ctx_is_auth_cb_set(he_ssl_ctx_t *ctx);
 void he_ssl_ctx_set_populate_network_config_ipv4_cb(
     he_ssl_ctx_t *ctx, he_populate_network_config_ipv4_cb_t pop_network_cb);
 
+/**
+ * @brief Sets te function that will be called when Lightway PMTUD changes state
+ * @param ctx A pointer to a valid SSL context
+ * @param pmtud_state_change_cb The function to be called when Lightway PMTUD changes state
```

Comment:
I briefly mentioned this when we discussed in your design doc, but I should have been more clear.  We've only explained `called when Lightway PMTUD changes state` here.  Can we explain a bit more about what the client is expected to do when the state changes?  I suppose we can explain it here, or in the header definition of `he_pmtud_state_t`.

Reply:
No, there's actually no expectation for the client to do anything. It **could** to set the DF flag on the socket when it receives state change callbacks, however it could also set the DF flag at the beginning of the connection and keep the DF flag set forever. So I don't think here is the right place to communicate the DF flag requirement.

I'll figure out another place for the overall PMTUD tech design doc.

Reply:
OK, then we should document that `There is no hard expectation for the client to do anything in this callback. However, this callback may be useful for optional optimizations triggered by stage changes`.  Otherwise, I think a new user of this callback would feel like they are missing something they should do, but don't know what it is.

Reply:
Done. But instead of here, I've added the documentation to the definition of `he_pmtud_state_change_cb_t`.